### PR TITLE
[SD-3150] - Priority added as a new search facet

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by akintolga** [comment]
Dependent on superdesk/superdesk-core#31

File: client/app/scripts/superdesk-search/views/search-facets.html [hunk 1]
```diff
@@ -74,6 +74,15 @@
             </ul>
         </div>
 
+        <div sd-toggle-box data-title="{{ 'Priority' | FacetLabels }}" ng-hide="isEmpty('priority')" data-open="true">
+            <ul class="filter-box">
+                <li ng-repeat="(key,value) in aggregations.priority" ng-click="toggleFilter('priority', key)">
+                    <span class="sd-checkbox" ng-checked="hasFilter('priority',key)"></span>
+                    <span>{{key | translate}} <i>{{ value }}</i></span>
```
Comment (plamut) [inline client/app/scripts/superdesk-search/views/search-facets.html]: We can probably use one-time bindings here, for both values?

**Comment by plamut** [comment]
Looks good, except for a small remark.

:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**